lib/hooks/useList.ts

```diff
@@ -107,6 +107,59 @@ const addListContributor = async (listId: string, contributors: number[]) => {
   }
 };
 
+interface ListContributorsHighlightsProps {
+  listId: string;
+  repo?: string;
+  range?: number;
+  initialData?: PaginatedListContributorsHighlightsResponse;
+  contributor?: string;
+  limit?: number;
+}
+
+export interface PaginatedListContributorsHighlightsResponse {
```

Comment:
This is a pattern that has been in the code base for sometime now... we might want to consider addressing this in a seperate PR since it is in other areas of the app.

Reply:
This is a pattern that has been in the code base for sometime now... we might want to consider addressing this in a seperate PR since it is in other areas of the app.